reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<p>The mass fraction of Argon here is zero, did you want to plot another species?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Typically Ar or N2 are added as a bath gas. I have now changed the initial composition to have 0.01% Ar in the mixture. Also there are just 4 gas-phase species in the mechanism and I am am plotting all of them. </p>